src/pages/home/contribution/Contribution.component.js

```diff
@@ -46,15 +48,28 @@ const Contribution = () => {
 
             <Styled.EngageCallBox isSmallScreen={isSmallScreen}>
               <ModuleTitle>{guideLang.engageCall}</ModuleTitle>
-              <img height={36} src={getImage('join-logos.svg')} alt={'join logos'} />
+              <Row gutter={16}>
+                <Col>
+                  <a href="https://internals.tidb.io/">
+                    <img height={36} src={getImage('asktug-logo.svg')} alt={'asktug logo'} />
+                  </a>
+                </Col>
+                <Col>
+                  <a href="https://slack.tidb.io/">
+                    <img height={36} src={getImage('slack-logo.svg')} alt={'slack logo'} />
+                  </a>
+                </Col>
+              </Row>
             </Styled.EngageCallBox>
           </>
         }
         rightPanel={
           <>
             <Styled.ForkTitle>{forkLang.title}</Styled.ForkTitle>
             <Styled.IssueList src={getImage('github-issues.jpg')} />
-            <img height={36} src={getImage('github-logo.svg')} alt={'github logo'} />
+            <a href={forkLang.link}>
```

Comment:
Please use `Link` under `src/component/link` because Link is based on [next/router](https://nextjs.org/docs/api-reference/next/router) and optimised for SPA navigations.

Reply:
I kept some native <a/> elements because <Link> is styled with blue font ant fixed font-size.
For most use cases, we're referencing external sources, and so SPA navigation aren't mandatory.